tasks/main.yaml

```diff
@@ -3,16 +3,39 @@
 - name: Test if distribution is RedHat
   assert:
     that: >
-      ansible_os_family == "RedHat" and
-      ansible_distribution_major_version | int >= 6
+      (ansible_os_family == "RedHat" and
+      ansible_distribution_major_version | int >= 6)
+      or (ansible_distribution == "Amazon" and
```

Comment:
As above, there is no need to list Amazon as supported distribution as it's based on RedHat which is already listed.